CV.html

```diff
@@ -1,69 +0,0 @@
-<!DOCTYPE html>
-<html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Portfolio</title>
-     <meta name="description" content="Interaktywny poradnik szybkiego startu dla Brackets.">
-        <link rel="stylesheet" href="portfolio.css">
-        <link href="https://fonts.googleapis.com/css?family=Pacifico|Poiret+One" rel="stylesheet">
-        
-               <script type="text/javascript">
-            
-            function odliczanie()
-            {
-                var data = new Date();
-                
-                var godzina = data.getHours();
-                if(godzina <10)
-                    {
-                        godzina="0"+godzina;
-                    }
-                var minuta = data.getMinutes();
-                if(minuta <10)
```

Comment:
I'd recommend formatting like if (minuta < 10) here, it's more readable that way